airflow/operators/hive_to_druid.py

```diff
@@ -97,7 +97,7 @@ def __init__(
         self.hadoop_dependency_coordinates = hadoop_dependency_coordinates
         self.druid_ingest_conn_id = druid_ingest_conn_id
         self.metastore_conn_id = metastore_conn_id
-        self.hive_tblproperties = hive_tblproperties
+        self.hive_tblproperties = hive_tblproperties or dict()
```

Comment:
```suggestion
        self.hive_tblproperties = hive_tblproperties or {}
```

Reply:
@feluelle apparently it's not just a matter or preference it's also a matter of performance. `{}` is much better:
https://stackoverflow.com/questions/17097985/dict-vs-in-python-which-is-better/17098017
https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better